### DISCUSSION TITLE
both Cyberduck and Mountainduck do not connect to my Redhat 8 server anymore ...

### DISCUSSION THREAD

**Discussion by heistera** [iterate-ch/cyberduck#14736] [Q&A] [answered]
Other connections work ... sftp -v arno@arnoserver does work ... no connection with both Cyberduck or Mountainduck possible ... please find attached the debug log from Cyberduck. Thanks! Arno
[cyberduck.log.zip](https://github.com/iterate-ch/cyberduck/files/11610807/cyberduck.log.zip)

**Comment by dkocher** [comment, marked as answer]
The log indicates you are trying to connect to `arnoserver` which is resolved to `192.168.1.128` which times out after 30000ms. You may want to attach the output from `sftp -v` from the command line.

**Comment by heistera** [comment]
Hi David! Thanks for the quick reply. With your comment I was able to identify my problem. My local arp cache, possibly other caches as well were out of date ... I solved it by rebooting my Fritz!Box at home. Thanks again and sorry for the noise! Cheers, Arno
